SpatialGDK/Documentation/content/get-started/gdk-template.md

```diff
@@ -0,0 +1,152 @@
+
+![The Unreal Engine Project Browser]({{assetRoot}}assets/set-up-template/template-project-browser.png)
+
+*Image: The Unreal Engine Project Browser, with the project file path and project name highlighted.*
+
+After clicking **Create Project**, the Unreal Engine generates the necessary project files before closing the editor and automatically opening Visual Studio. 
+
+### Clone the GDK
+
+Now you need to clone the SpatialOS GDK for Unreal into your project.
+
+1. In **File Explorer**, Navigate to the `<Game>` directory and create a `Plugins` folder in this directory.
+2. In a Git Bash terminal window, navigate to `<Game>\Plugins` and clone the [GDK for Unreal](https://github.com/spatialos/UnrealGDK) repository by running either:
+    * (HTTPS) `git clone https://github.com/spatialos/UnrealGDK.git`
+    * (SSH) `git clone git@github.com:spatialos/UnrealGDK.git`
+
+>**Note** You need to ensure that the root directory of the GDK for Unreal repository is called `UnrealGDK` so the file path is: `<Game>\Plugins\UnrealGDK\...`
+
+### Build the dependencies 
+
+To use the starter template, you must build the GDK for Unreal module dependencies and add the GDK to your project. 
+
+1. Open **File Explorer**, navigate to the root directory of the GDK for Unreal repository, and double-click **`Setup.bat`**. You may be prompted to sign into your SpatialOS account if you have not already.
+1. In the same directory, double-click **`<YourProject>`.sln** to open it with Visual Studio.
+1. In the Solution Explorer window, right-click on **`<YourProject>`** and select **Build**.
+1. Open **`<YourProject>`.uproject** in the Unreal Editor. 
+1. Select the [**Schema**](https://docs.improbable.io/reference/latest/shared/glossary) GDK for Unreal Toolbar button to generate schema 
+    
+    ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/schema-button.png)<br/>
+    _Image: On the GDK toolbar in the Unreal Editor select the **Schema** button_<br/>
+
+1. Select the [**Snapshot**]({{urlRoot}}/content/generating-a-snapshot) toolbar button to generate a snapshot.
+
+    ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/snapshot-button.png)<br/>
+    
+    _Image: On the GDK toolbar in the Unreal Editor select the **Snapshot** button_
+
+
+### Deploy your project 
+
+#### Deploy locally
+
+1. In the Unreal Editor, on the SpatialOS GDK toolbar, select **Start**. This opens a terminal window and starts a local SpatialOS deployment. Wait until you see the output `SpatialOS ready. Access the inspector at http://localhost:21000/inspector` in your terminal window.<br/>
+![Toolbar]({{assetRoot}}assets/set-up-template/template-start.png)<br/>
+_Image: On the GDK toolbar in the Unreal Editor select the **Start** button_<br/>
+1. On the Unreal Editor toolbar, open the **Play** drop-down menu.
+1. Under **Multiplayer Options**, set the number of players to **2** and check the box next to **Run Dedicated Server**. Then, under Modes, select **New Editor Window (PIE)**.<br/>
```

Comment:
Should "Modes" be bolded? I can't tell? Probably yes as we have 'Multiplayer Options' bold?

Reply:
It should and now is. Also edited the screenshot to better highlight the **New Editor Window (PIE)** option.